README.md

```diff
@@ -0,0 +1,24 @@
+# Overview
+
+This library contains tools for unit testing layered charms.
```

Comment:
Note: this README is really out of date. Will fix.

In the meantime, the docstring in the Harness class has up-to-date info on how this all works.